main.py

```diff
@@ -5,6 +5,6 @@ def __init__():
 def main():
     print("Let's get started!")
     choice = int(input("Please choose from the following\n1. GitHub\n2. Facebook\n3. Twitter\n")) # Add more choices here
-
+    c = a + b # lets do some computation
```

Comment:
is `a` and `b` defined?

Reply:
since only one line of code change was allowed, i had to assume that somebody else will define it.